site/en/performance/_toc.yaml

```diff
@@ -0,0 +1,31 @@
+toc:
+- title: Performance
+  path: /performance/
+- title: Performance guide
+  path: /performance/performance_guide
+- title: Input pipeline performance guide
+  path: /performance/datasets_performance
+- title: Benchmarks
+  path: /performance/benchmarks
+
+- heading: Model optimization Toolkit
```

Comment:
I'm still unclear what the "Model optimization toolkit" is? Is it a product or techniques? Should this be "Model optimization methods"? (or something) Is there something else to download, install, import, etc?